examples/window-app/common/src/WindowApp.cpp

```diff
@@ -109,10 +120,26 @@ WindowApp::Cover * WindowApp::GetCover(chip::EndpointId endpoint)
     return nullptr;
 }
 
+CHIP_ERROR WindowApp::StartAppTask() { return CHIP_NO_ERROR;}
```

Comment:
This function reference is needed for the below design, otherwise getting build error.  
Called StartApptask function from the main() which will create AppTaskMain task and the Event Queue.
The AppTaskMain will initializing the window application and run the event handler loop.Removed the redundant task exist before and now having only one task and the flow is similar for both WiFi and Thread. 
The design flow is similar to the Light and door-lock apps except the base application class reference.

Reply:
@jepenven-silabs  In main function calling startAppTask() through window object called "app" hence we declare and define virtual startAppTask() in windowApp.h and windowApp.cpp file respectively

If removed it, will get linking error

Reply:
As stated above.